pdc/apps/component/tests.py

```diff
@@ -1866,6 +1866,32 @@ def test_delete_group(self):
         response = self.client.get(url, format='json')
         self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
 
+    def test_list_relationship_by_nested(self):
+
+        def f(ordering_value):
+            response = self.client.get(reverse("rcrelationship-list"), {'ordering': ordering_value}, format='json')
+
+            self.assertEqual(response.status_code, status.HTTP_200_OK)
+            self.assertEqual(response.data.get('count'), 2)
+            data = response.data.get('results')
+            if ordering_value == "to_component__name":
+                self.assertTrue(self, data[0].get('to_component')["name"] <= data[1].get('to_component')["name"])
+            elif ordering_value == "-to_component__name":
+                self.assertTrue(self, data[0].get('to_component')["name"] >= data[1].get('to_component')["name"])
+            elif ordering_value == "to_component__release":
+                self.assertTrue(self, data[0].get('to_component')["release"] <= data[1].get('to_component')["release"])
+            else:
+                self.assertTrue(self, data[0].get('to_component')["release"] >= data[1].get('to_component')["release"])
+
+        for v in ["to_component__name", "-to_component__name", "to_component__release", "-to_component__release"]:
+            f(v)
```

Comment:
@hluk  I added a new test function here, called test_list_relationship_multi_nested, which ordered by  to_component__release,to_component__name and -to_component__name,to_component__release.
Do you mean that?